website/src/pages/showcase.js

```diff
@@ -77,13 +77,13 @@ const Showcase = () => {
   const {siteConfig} = useDocusaurusContext();
 
   const {meta, microsoft, shopify, wix, others} = siteConfig.customFields.users;
-  const [pinnedRandomizedApps, setPinnedRandomizedApps] = useState([]);
-  const [randomizedApps, setRandomizedApps] = useState([]);
 
-  useEffect(() => {
-    setRandomizedApps(randomizeApps(others.filter(app => !app.pinned)));
-    setPinnedRandomizedApps(randomizeApps(others.filter(app => app.pinned)));
-  }, []);
+  const [pinnedRandomizedApps] = useState(
+    randomizeApps(others.filter(app => !app.pinned))
+  );
+  const [randomizedApps] = useState(
+    randomizeApps(others.filter(app => app.pinned))
+  );
```

Comment:
Removing `useEffect` wrap for entries randomization, leads to the rendering issue, where app labels do not match descriptions. This was written that way to address those issues.

 
<img width="1016" alt="Screenshot 2023-03-14 090331" src="https://user-images.githubusercontent.com/719641/224935113-410b88d0-1c52-4947-87f8-ca2a1509cf54.png">